INSTALL.md

```diff
@@ -69,40 +69,36 @@ brew info elasticsearch
 ```
 
 ### MYSQL Database
-Ensure your database is running.
+Start MYSQL with the following command:
 ```
 mysql.server start
 ```
 
-Then run mysql creation script from root directory:
+Then run the MYSQL creation script from project root directory:
 ```
 ./create-mysql-db.sh
 ```
-If you would like to have a custom db setup then you can pass in the necessary args.
-> **NOTE** be sure to update your local settings `local.py` to account for these changes.
+
+If you would like to have a custom database setup then you can pass in the necessary arguments:
 
 ```
 ./create-mysql-db.sh <dbname> <username> <password>
 ```
 
+> **NOTE:** Be sure to update your local settings in
+  `cfgov/cfgov/settings/local.py` to account for these changes.
 
 
-### Sheer-like & Sheer
-> **NOTE** Ensure you install Sheer-like first to avoid version conflicts with dependencies.
-
-To [install Sheer-like](https://github.com/cfpb/django-sheerlike/blob/master/README.rst), start by cloning the
-Sheer GitHub project to wherever you keep your projects (not inside cfgov-refresh directory):
+### Sheer-like
+To [install Sheer-like](https://github.com/cfpb/django-sheerlike/blob/master/README.rst),
+start by cloning the Sheer-like GitHub project to wherever you keep your projects
+(not inside cfgov-refresh directory):
```

Comment:
Updated in https://github.com/cfpb/cfgov-refresh/commit/72f64ba14c5b3573e82fb6b160803f0dcbc8d8a7 Please check that this is correct.